### DISCUSSION TITLE
Dynamic, Persistent buttons

### DISCUSSION THREAD

**Discussion by mooses82** [nextcord/nextcord#1231] [Q&A]
Hello,
I am looking to make buttons persistent and dynamic on my bot written in Nextcord. I've been look all over, but cannot seem to find a tutorial on how to do this. If anyone is able to give a "tutorial" on what to do, that would be appreciated. I am fine to use a Database if needed to make them still work after my bot restarts. 

Thanks

**Comment by DenverCoder1** [comment]
There's example code here:
https://github.com/nextcord/nextcord/blob/master/examples/views/persistent.py

This video tutorial I made may also help: 
https://youtu.be/jMeOejBy8Hc?si=FctPhGIkfFxXS407

**Reply by mooses82** [reply]
I will check this out later, Thank you for the response!
